R/assertions.R

```diff
@@ -1252,11 +1255,14 @@ assert_varval_list <- function(arg,
   if (accept_expr) {
     valid_vals <- "a symbol, character scalar, numeric scalar, an expression, or `NA`"
   }
+  else if (accept_var){
+    valid_vals <- "a symbol, character scalar, numeric scalar, variable names or `NA`"
+  }
   else {
     valid_vals <- "a symbol, character scalar, numeric scalar, or `NA`"
   }
 
-  if (!is_quosures(arg) || !is_named(arg)) {
+  if (!accept_var & (!is_quosures(arg) || !is_named(arg))) {
```

Comment:
Unnamed elements should be accepted only if it is a symbol, e.g., `l <- vars(USUB, SEQ = ASEQ, 1)` should not be accepted.

Reply:
Updated